doc/source/comparison_with_r.rst

```diff
@@ -55,7 +55,7 @@ R                                            pandas
 ``select(df, col1, col2)``                   ``df[['col1', 'col2']]``
 ``select(df, col1:col3)``                    No one-line equivalent, but see [#select_range]_
 ``select(df, -(col1:col3))``                 ``df.drop(cols_to_drop, axis=1)`` but see [#select_range]_
-``distinct(select(df, col1))``               ``df.col1.unique()``
+``distinct(select(df, col1))``               ``df[['col1']].drop_duplicates()``
```

Comment:
hmm that's interesting. ok best then to show the frame result then (which i think is what you did) (even for 1 column)